pkg/operatorsource/helper_test.go

```diff
@@ -88,3 +92,35 @@ func helperNewCatalogSourceConfigWithLabels(namespace, name string, opsrcLabels
 
 	return csc
 }
+
+func helperNewFakeClient() client.Client {
+
+	scheme := runtime.NewScheme()
+	apis.AddToScheme(scheme)
+	return fake.NewFakeClientWithScheme(scheme)
+
```

Comment:
The space before and after the code block in this function seems unnecessary.

Reply:
@kevinrizza the previous code blocks in this file have one space before and one after each. Won't it be inconsistent to not introduce spaces around any new code block we add?

Reply:
Also, cleaned up noise from commit.

Reply:
Well they are inconsistent. Some have a space above the code block (but not all), but as far as I see none of them have a space after.

Reply:
Please don't add newlines at the beginning and end of every function. It makes code looks verbose and is a bad precedent to set.

Reply:
@kevinrizza I see your point. Removing the spaces.